src/settings/LocationLocations/LocationForm/LocationFormContainer.js

```diff
@@ -85,12 +86,14 @@ const LocationFormContainer = ({
   }, [onSave, servicePointsByName, saveData]);
 
   return (
-    <LocationForm
-      {...rest}
-      parentMutator={parentMutator}
-      initialValues={initialValues}
-      onSubmit={saveLocation}
-    />
+    <TitleManager page={intl.formatMessage({ id: 'ui-tenant-settings.settings.location.creatNew.title' })}>
```

Comment:
Typo: `creatNew` should be `createNew`, or even just `create`.

Reply:
done! Thanks for notions